.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,4 @@
+name: "CodeQL config"
+
+paths-ignore:
+  - vendor/assets
```

Comment:
We can add the bgeigie viewer to this list as well. It's sourced from https://github.com/Safecast/Tilemap/blob/master/bgeigie_viewer.js but I didn't find a better way to include it than just checking it in.